examples/with-redux-observable/pages/index.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 import Link from 'next/link'
 import withRedux from 'next-redux-wrapper'
-import initStore from '../lib'
-import { startFetchingCharacters, stopFetchingCharacters } from '../lib/reducer'
-import * as api from '../lib/api'
+import initStore from '../redux'
 import CharacterInfo from '../components/CharacterInfo'
+import { rootEpic } from "../redux/epics";
+import * as actions from "../redux/actions";
+import { of } from "rxjs/observable/of";
```

Comment:
I think this is going to import whole `rxjs` library, isn't it?

Reply:
No, this only imports the static creation operator "of".

Reply:
Yes, I got confused because I'm using [a library](https://github.com/mcmunder/universal-rxjs-ajax/blob/master/src/index.js) that I didn't know is importing the whole package.